activiti-cloud-services-org/activiti-cloud-services-org-rest/src/test/resources/process-extensions/valid-extensions.json

```diff
@@ -27,32 +27,39 @@
     },
     "properties": {
       "c297ec88-0ecf-4841-9b0f-2ae814957c68": {
-        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c68",
+        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c48",
         "name": "integerVariable",
         "type": "integer",
         "required": false,
         "value": 12
       },
       "c297ec88-0ecf-4841-9b0f-2ae814957c69": {
-        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c68",
+        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c38",
         "name": "stringVariable",
         "type": "string",
         "required": false,
         "value": "aloha"
       },
       "c297ec88-0ecf-4841-9b0f-2ae814957c62": {
-        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c68",
+        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c28",
         "name": "dateVariable",
         "type": "date",
         "required": false,
         "value": "1993-06-03"
       },
       "c297ec88-0ecf-4841-9b0f-2ae814957c64": {
-        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c68",
+        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c18",
         "name": "booleanVariable",
         "type": "boolean",
         "required": false,
         "value": true
+      },
+      "c297ec88-0ecf-4841-9b0f-2ae814957c67": {
+        "id": "c297ec88-0ecf-4841-9b0f-2ae814957c08",
+        "name": "objectVariable",
+        "type": "object",
```

Comment:
According with https://github.com/Activiti/Activiti/issues/2500, expected value for "type" is "json"